website/src/data/gallery.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "title": "AutoGen Playground",
+    "link": "https://huggingface.co/spaces/thinkall/AutoGen_Playground",
+    "description": "A space to explore the capabilities of AutoGen.",
+    "tags": ["ui", "app"]
+  },
+  {
+    "title": "Solving Security Vulnerabilities",
+    "link": "https://www.linkedin.com/pulse/solving-security-vulnerabilities-llms-society-mind-model-leah-bonser-kcswc?trk=public_post_feed-article-content",
+    "description": "An article discussing the use of AutoGen to tackle security vulnerabilities.",
+    "image": "default.png",
+    "tags": ["app"]
```

Comment:
Is it an app?
Can you define what each tag means somewhere? There is a long list of examples here and it'll take a long time to iterate like this.

Reply:
Done. It is hard to choose the tag for each demo since it is subjective and some tags like "app" could portray most demos. So I say "We recommend using no more than two tags for clarity." in the Contribution guide.